lib/ncr/mailboxes.rb

```diff
@@ -0,0 +1,29 @@
+module Ncr
+  module Mailboxes
+    def self.ba61_tier1_budget
+      self.approver_with_role("BA61_tier1_budget_approver")
+    end
+
+    def self.ba61_tier2_budget
+      self.approver_with_role("BA61_tier2_budget_approver")
+    end
+
+    def self.ba80_budget
+      self.approver_with_role("BA80_budget_approver")
+    end
+
+    def self.ool_ba80_budget
+      self.approver_with_role("OOL_BA80_budget_approver")
+    end
+
+    protected
+
+    def self.approver_with_role(role_name)
```

Comment:
When I see `protected` as a developer, I think "something that inherits from this class uses these methods` when I see `private`, I think: this class itself uses these methods. Agreed that it is an indication to other developers, but there are also real differences in behavior so I think it is worth using private/protected as intended. See http://tenderlovemaking.com/2012/09/07/protected-methods-and-ruby-2-0.html

Interesting that you see this module as a singleton...I usually think of modules in ruby as a set of behaviors to be mixed in multiple other classes. Since we are just using this in one other class, that is not how we are using this set of methods. 

Agreed that there is not a conceptual reason for us to have multiple instances. I would still LOVE to change up a lot of the underling behavior with these mailboxes since our seeds dependency / slow test suite is really cramping my TDD style...so having these methods extracted into a separate module will make it easier to do further changes down the road, I hope!